test/optim/test_optim.py

```diff
@@ -1,6 +1,11 @@
 # Owner(s): ["module: optimizer"]
 
+from __future__ import annotations
```

Comment:
Nit: Is there a more global way to enable this across the repo? Vs. adding it everywhere?

Reply:
I looked online and couldn't find anything -- I'll start an issue on CPython for it

Reply:
for the purposes of this pr this is fine + you do not have to but ofc it is up to u!